src/GitExtensions2022/source.extension.cs

```diff
@@ -11,7 +11,7 @@ static class Vsix
         public const string Name = "GitExtensions";
         public const string Description = @"Git Extensions is a graphical user interface for Git that allows you to control Git without using the command-line";
         public const string Language = "en-US";
-        public const string Version = "2.50.02";
+        public const string Version = "2.51.0";
```

Comment:
Why is this getting changed?

Reply:
I added this for local testing. VS threw error saying i had the extension installed. I will remove this change.